share/completions/stow.fish

```diff
@@ -0,0 +1,25 @@
+# Sources:
+# stow --help 
+# https://www.gnu.org/software/stow/manual/stow.html
+# https://git.savannah.gnu.org/cgit/stow.git/tree/NEWS
+
+# options
+complete -c stow -s d -l dir -r -d 'Set stow dir, default $STOW_DIR or current dir'
+complete -c stow -s t -l target -r -d 'Set target dir, default parent of stow dir'
+complete -c stow -l ignore -x -d 'Ignore files ending in this Perl regex'
+complete -c stow -l defer -x -d "Don't stow files beginning with this Perl regex if already stowed to another package"
+complete -c stow -l override -x -d "Force stowing files beginning with this Perl regex if already stowed to another package"
+complete -c stow -l no-folding -d "Create dirs instead of symlinks to whole dirs"
+complete -c stow -l adopt -d "CAUTION, OVERWRITES FILES IN STOW DIR! Move existing files into stow dir if target exists"
```

Comment:
No need to have the warning that long:

"Move existing files into stow dir if target exists (and overwrite!)"

Reply:
ok, I implemented the other suggestions without modifications, but for this one I have allcapsed the `and overwrite`, since the documentation is very insistent that this is exceptional behaviour, since it's the only option that can make stow lose data.